### ISSUE TITLE
Official Flatpak Release

### ISSUE THREAD

**Issue by ToasterUwU** [OscToys/OscGoesBrrr#13]
In https://github.com/OscToys/OscGoesBrrr/blob/main/package.json#L68-L72 there already is a thing configured to build an AppImage, and when using electron-builder to build that AppImage on Linux, it works fine and the resulting AppImage also works fine. 
So it would only make sense to provide that AppImage as a part of each Release. 

If you want more testing from a Linux user before you throw it out in the world, I'm happy to help.

**Comment by ToasterUwU** [comment]
Of course, a Flatpak published on Flathub would be way better, but im not sure if Flathub allows NSFW things

**Comment by ToasterUwU** [comment]
I tried building a Flatpak, and it worked. Installing it from the file and using it works fine as well. 
I also checked if Flathub allows NSFW Software, and they never mention it as a potential problem. As long as its legal, its allowed.

So i highly recommend going with a Flatpak Release instead of AppImage, since its a universal standard that works on every kind of Linux, while also being more than a portable app, so it can also be downloaded with Package Managers instead of downloading it by hand. 
Happy to help however i can.

**Comment by ToasterUwU** [comment]
@doomsdayrs

Hey, thanks for making such rapid progress on the intiface flathub release PR, that's really nice.

Would you be willing to help me out with this one? I have never made a flathub release, but I got the support from @SenkyDragon on discord for making this happen. All they want is total control over it in the end, to make sure there is no 3rd party doing any funny business.

This plus intiface would make the software chain complete and truely cross platform, so that would be awesome.
I would totally understand if you say you don't care about this software and don't wanna help.
Asking is worth the shot I think. I'm sure I can eventually figure it out on my own, but you seem to know what you are doing with intiface, so help from you would make this way easier and faster I think.

**Comment by Doomsdayrs** [comment]
Sure, but I do not have time on my plate this week for this. Maybe next week I will have more time.

**Comment by ToasterUwU** [comment]
@Doomsdayrs Thanks a lot. 

I will start working on this thing as much as i can, and if i get stuck i will ping you. You look at that stuff whenever you get the time and will for it, no pressure.

**Comment by ToasterUwU** [comment]
Actually, nvm. I kinda have other things on my list, and the process kinda overwhelms me from looking at the details.. 
So i will wait for you, im happy to help however i can, but i think you have to do the initial stuff

**Comment by Doomsdayrs** [comment]
@SenkyDragon I will need an SVG for this.

**Comment by Doomsdayrs** [comment]
Here is a not-tested branch for the work.
https://github.com/Doomsdayrs/flathub/tree/toys.osc.oscgoesbrrr

**Comment by Doomsdayrs** [comment]
@SenkyDragon Also, what are you build instructions?

Because it seems you do not use [electron-builder](https://github.com/electron-userland/electron-builder)?

**Comment by ToasterUwU** [comment]
@Doomsdayrs I used electron builder for the test. So either way, you can use that for the flathub for sure

**Comment by SenkyDragon** [comment]
> @SenkyDragon Also, what are you build instructions?
> 
> Because it seems you do not use [electron-builder](https://github.com/electron-userland/electron-builder)?

You can see the full deployment process here. electron-builder is used: https://github.com/OscToys/OscGoesBrrr/blob/main/.github/workflows/deploy.yml#L67

This workflow is where the deployment process to flathub would need to happen as well.

**Comment by SenkyDragon** [comment]
There is no SVG available for the logo, so it'd have to be re-traced. The font is just Arial.

**Comment by ToasterUwU** [comment]
@Doomsdayrs @SenkyDragon

Here is an SVG of the Logo. If you need me to change anything about it, let me know

![ogb-logo](https://github.com/OscToys/OscGoesBrrr/assets/43654377/e4f0bc17-03e7-451b-b551-f21440d7a899)

**Comment by ToasterUwU** [comment]
@Doomsdayrs Need anything else?

**Comment by ToasterUwU** [comment]
@Doomsdayrs I posed the logo above, in case you missed it.

Need anything else? Happy to help however i can
